config/routes.rb

```diff
@@ -1,6 +1,15 @@
 Rails.application.routes.draw do
+  
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
   # Defines the root path route ("/")
   # root "articles#index"
+  resources :users, only: [:index, :show] do 
+    resources :posts, only: [:index, :show, :new, :create]
+  end
```

Comment:
- [ ] You're doing a really great job so far but you seem to have missed one minor detail from the project requirements, you should only expose the 4 required URLs for the time being which are the `user#index`, `user#show`, `posts#index` & `posts#show` so kindly make sure to remove the posts `new` and `create` endpoints.
- [ ] After making the change and running `rails routes | grep user` you should only see the below output:
  ![](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/blog-app/images/routes.png?token=GHSAT0AAAAAACJTIBN6L4WT5Y24TAM3FF5QZJ7ONZA)